lessons/specifics/plug.md

```diff
@@ -6,13 +6,20 @@ order: 1
 lang: en
 ---
 
-If you're familiar with Ruby you can think of Plug as Rack with a splash of Sinatra.  It provides a specification for web application components and adapters for web servers. While not part of Elixir core, Plug is an official Elixir project.
+If you're familiar with Ruby you can think of Plug as Rack with a splash of Sinatra.
+It provides a specification for web application components and adapters for web servers.
+While not part of Elixir core, Plug is an official Elixir project.
+
+We'll start by creating a minimal Plug-based web application.
+After that, we'll learn about Plug's router and how to add a Plug to an existing web application.
 
 {% include toc.html %}
 
 ## Installation
 
-Installation is a breeze with mix.  To install Plug we need to make two small changes to our `mix.exs`.  The first thing to do is add both Plug and a web server (we'll be using Cowboy) to our file as dependencies:
+Installation is a breeze with mix.
+To install Plug we need to make two small changes to our `mix.exs`.
+The first thing to do is add both Plug and a web server (we'll be using Cowboy) to our file as dependencies:
```

Comment:
file --> Mixfile would sound better IMO, as there will obviously be many files in a project and this seems ambiguous